Composer/packages/server/src/services/storage.ts

```diff
@@ -96,6 +96,7 @@ class StorageService {
   public updateCurrentPath = (path: string) => {
     this.storageConnections[0].path = path;
     Store.set(this.STORE_KEY, this.storageConnections);
+    return this.storageConnections;
```

Comment:
I am okay to return this.storageConnections.

But I think you don't need to return this, because the caller should be able to grab it by themselves.

The better approach, which may not applies to this code, is that. If you have an update function that modify a value and return it out. It is better to treat the value immutable. That means, the function get passed in, will not be changed. The returned value will be a new one with modified content.